app/user/[id]/[postType]/@user/page.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { Suspense } from 'react';
+import { getProfileHeader } from '@/app/actions/profile';
+import Loading from './loading';
+import ProfileHeader from '@/src/compositions/profile-header/profile-header';
+
+export default async function UserProfile({
+    params,
+}: {
+    params: { id: string; postType: string };
+}) {
+    const { user, isActiveUser } = await getProfileHeader(params.id);
+
+    return (
+        <Suspense fallback={<Loading />}>
+            <ProfileHeader user={user} activeUser={isActiveUser} />
```

Comment:
here we don't need the profile header, but up to 6 random users - but we can implement that in a separate branch, since it's a new feature